src/Forms/GroupForm/GroupForm.jsx

```diff
@@ -140,6 +142,10 @@ const GroupForm = ({
     );
   }, [languagePackages]);
 
+  useEffect(() => {
+    onLoad && onLoad();
+  }, [onLoad]);
```

Comment:
Is this intended, That the onload will retrigger if the function changes?

Reply:
I can't remember I wrote this function. Maybe @zikowang knows more?

Reply:
> Beim onLoad && onLoad() ist es my bad. Da sollte das dependency array des useEffects leer sein.
@zikowang on discord.

Reply:
done